### PR TITLE
add refresh token instructions and fix typos doc88783

### PR DESCRIPTION
**Pull request by mystiberry-okta**
@tthompson-okta or @karlmcguinness-okta I fixed two typos and added clarifying info about refresh token use. Is this the best place for the new info?

### CODE REVIEW

File: _source/docs/api/resources/oidc.md [hunk 1]
```diff
@@ -374,6 +374,13 @@ For authentication with Basic auth, an HTTP header with the following format mus
 Authorization: Basic ${Base64(<client_id>:<client_secret>)} 
 ~~~
 
+#####Refresh Tokens
+
+You can use refresh tokens if you create a mobile or web OIDC app. To use refresh tokens:
+
+* Check the <b>Allow Offline Access</b> checkbox in Okta to enable refresh tokens.
```
Comment (tthompson-okta) [inline _source/docs/api/resources/oidc.md]: I'd reword to: `Check the <b>Allow Offline Access</b> checkbox on the client application page in the Okta Administrative UI (this option is only visible for WEB and NATIVE application types).`

File: _source/docs/api/resources/oidc.md [hunk 2]
```diff
@@ -374,6 +374,13 @@ For authentication with Basic auth, an HTTP header with the following format mus
 Authorization: Basic ${Base64(<client_id>:<client_secret>)} 
 ~~~
 
+#####Refresh Tokens
```
Comment (tthompson-okta) [inline _source/docs/api/resources/oidc.md]: This should actually go up next to the "Authorization Endpoint" rather than here by the "Token Endpoint". Also, we need to update the description for the `scope` parameter to list "offline_access" as a valid scope.

**Comment by mystiberry-okta** [comment]
I'm just waiting on these until we get the doc restructure merged. @royalchan-okta  if you don't think it'll cause a conflict, I can go ahead and fix these now. What would you advise?

**Comment by mystiberry-okta** [comment]
Because the file was split, conflict resolution is making my head hurt. Closing this PR and opening a new one with Trevor's comments implemented. Since the file split up, probably worth a second look anyway.
